eqcorrscan/tests/correlate_test.py

```diff
@@ -2,108 +2,399 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
-import unittest
-import numpy as np
+
+import copy
+import itertools
 import time
+import unittest
+from collections import defaultdict
+from functools import wraps
 
+import eqcorrscan.utils.correlate
+import eqcorrscan.utils.correlate as corr
+import numpy as np
+import pytest
+from eqcorrscan import register_array_xcorr
```

Comment:
With the loss of the direct import this needs to be reverted back to:
```python
from eqcorrscan.utills.correlate import register_array_xcorr
```

Reply:
right you are. That is what I get for assuming a small change wont break everything :)